swap_meet/decor.py

```diff
@@ -1,2 +1,8 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, category="Decor", condition=0):
```

Comment:
I'd echo the concern above about allowing `category` as a parameter to the constructor